notebooks/MAST/HSC/CASJOBS/HCV_API/HCV_API_demo.ipynb

```diff
@@ -0,0 +1,1039 @@
+    "\n",
+    "This notebook is available for [download](HCV_API_demo.ipynb)."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Instructions: \n",
+    "* Complete the initialization steps [described below](#initialization).\n",
+    "* Run the notebook to completion. \n",
+    "* Modify and rerun any sections of the Table of Contents below.\n",
+    "\n",
+    "Running the notebook from top to bottom takes less than 1 minute (depending on the speed of your computer and network connection).\n",
+    "\n",
+    "# Table of Contents\n",
+    "* [Intialization](#initialization)\n",
+    "* [Variable objects in IC 1613](#ic1613)\n",
+    "    * [Name resolver](#resolver)\n",
+    "    * [Select objects from HCV](#summary)\n",
+    "    * [Information on HCV variable classification](#classification)\n",
+    "    * [Sky coverage](#positions)\n",
+    "    * [Properties of variable objects](#variability)\n",
+    "    * [Color magnitude diagram](#cmd)\n",
+    "* [Light curve for a nova in M87](#m87)\n",
+    "    * [Extract and plot light curve for the nova](#lightcurve)\n",
+    "    * [HLA cutout images for selected measurements](#cutouts)\n",
+    "* [Compare the HCV automatic classification to expert validations](#expert)\n",
+    "    * [Plot MAD variability index distribution](#mad_expert)\n",
+    "    * [Plot fraction of artifacts vs. MAD](#artifacts)\n",
+    "* [Plot light curve for most variable high-quality candidate in the HCV](#most_variable)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Initialization <a name=\"initialization\"></a>"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Install Python modules\n",
+    "\n",
+    "_This notebook requires the use of **Python 3**._\n",
+    "\n",
+    "This needs some special modules in addition to the common requirements of `astropy`, `numpy` and `scipy`.  For anaconda versions of Python the installation commands are:\n",
+    "\n",
+    "<pre>\n",
+    "conda install requests\n",
+    "conda install pillow\n",
+    "</pre>\n",
+    "\n",
+    "Run the commands one at a time since conda may ask for confirmation."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "%matplotlib inline\n",
+    "import astropy\n",
+    "import time\n",
+    "import sys\n",
+    "import os\n",
+    "import requests\n",
+    "import json\n",
+    "import numpy as np\n",
+    "import matplotlib.pyplot as plt\n",
+    "\n",
+    "from PIL import Image\n",
+    "from io import BytesIO\n",
+    "\n",
+    "from astropy.table import Table, join\n",
+    "from astropy.io import ascii\n",
+    "\n",
+    "# Set page width to fill browser for longer output lines\n",
+    "from IPython.display import display, HTML\n",
+    "display(HTML(\"<style>.container { width:100% !important; }</style>\"))\n",
```

Comment:
This does not work reliably in the present day - e.g. jupyterlab doesn't respect it.  Your call whether you still want to keep it, but I'd probably drop it.